src/teamweek.ts

```diff
@@ -170,34 +179,38 @@ export function getMe(): TaskEither<unknown, User> {
 }
 
 export function getBuildoWorkspace(): TaskEither<unknown, Workspace> {
-  return getMe().chain(user =>
-    array
-      .findFirst(user.workspaces, w => w.name.toLowerCase() === "buildo")
-      .fold(fromLeft("'buildo' workspace not found"), taskEither.of)
+  return taskEither.chain(getMe(), user =>
+    fold(
```

Comment:
🤔 some functions come from instance dictionaries, some functions are "free".
I know the technical reason, but the API feels arbitrary and I constantly wonder: "do I search for this in the instance or is it a free function?".

At the very beginning, I've used the free `chain` function, which obviously didn't compile. Then I've spent a good minute or two searching for `option.fold` which doesn't exist.